src/platform_impl/web/web_sys/event.rs

```diff
@@ -246,3 +246,11 @@ pub fn codepoint(event: &KeyboardEvent) -> char {
     // https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key
     event.key().chars().next().unwrap()
 }
+
+pub fn touch_position(event: &PointerEvent) -> LogicalPosition<f64> {
+    LogicalPosition {
+        x: event.client_x() as f64,
+        // Flip the Y axis because canvas's origin is top-left.
```

Comment:
This comment should probably be removed? There is no flipping going on here.

```suggestion
```

Reply:
Thanks, I've restored flipping.

Reply:
Maybe we should make flipping optional? I don't seem to need it flipped (or have a setting to skip flipping for now until later when we can correctly identify all the exact conditions when we should flip).